concepticondata/conceptlists/DellaRosa-2010-417.tsv

```diff
@@ -0,0 +1,418 @@
+DellaRosa-2010-417-228	228	Barba	beard	678	BEARD	685	684	576	191	631	110	189
+DellaRosa-2010-417-229	229	Bicchiere	glass			669	696	695	150	694	106	128
+DellaRosa-2010-417-230	230	Bicicletta	bicycle	747	BICYCLE	700	697	653	176	662	106	325
+DellaRosa-2010-417-231	231	Bistecca	steak			689	697	684	197	671	113	350
+DellaRosa-2010-417-232	232	Bottiglia	bottle	1321	BOTTLE	697	700	700	185	654	105	138
+DellaRosa-2010-417-233	233	Braccio	arm	1673	ARM	697	679	680	200	646	105	135
+DellaRosa-2010-417-234	234	Brezza	breeze	3982	BREEZE	529	547	447	419	565	279	280
+DellaRosa-2010-417-235	235	Cacciatore	hunter	1053	HUNTER	655	647	494	229	611	129	325
+DellaRosa-2010-417-236	236	Calzino	sock	1522	SOCK	691	680	618	185	669	105	121
+DellaRosa-2010-417-237	237	Cammello	camel	1871	CAMEL	695	676	537	224	626	131	488
+DellaRosa-2010-417-238	238	Campeggio	camp	1105	CAMP	604	657	500	325	638	197	341
+DellaRosa-2010-417-239	239	Cane	dog	2009	DOG	700	700	647	163	687	117	115
+DellaRosa-2010-417-240	240	Canoa	canoe	1970	CANOE	679	688	440	335	600	114	288
+DellaRosa-2010-417-241	241	Canzone	song	1096	SONG	544	464	670	179	631	216	200
+DellaRosa-2010-417-242	242	Cappello	hat	771	HAT	669	687	579	169	668	115	156
+DellaRosa-2010-417-243	243	Carciofo	artichoke	3997	ARTICHOKE	691	676	579	282	656	105	142
+DellaRosa-2010-417-244	244	Carota	carrot	3085	CARROT	700	671	668	179	650	100	194
+DellaRosa-2010-417-245	245	Casco	helmet	1680	HELMET	688	680	591	288	644	110	226
+DellaRosa-2010-417-246	246	Cavallo	horse	615	HORSE	679	692	585	182	675	100	163
+DellaRosa-2010-417-247	247	Caverna	cavern			623	678	405	231	641	141	374
+DellaRosa-2010-417-248	248	Cellula	cell			555	476	461	471	488	165	595
+DellaRosa-2010-417-249	249	Cemento	cement			677	632	538	300	625	106	356
+DellaRosa-2010-417-250	250	Cenere	ash	646	ASH	681	679	441	260	538	138	236
+DellaRosa-2010-417-251	251	Cera	wax			674	588	500	282	547	144	350
+DellaRosa-2010-417-252	252	Chiesa	church	1349	CHURCH	636	650	638	225	655	150	288
+DellaRosa-2010-417-253	253	Ciliegie	cherry	2765	CHERRY	700	671	656	192	660	103	194
+DellaRosa-2010-417-254	254	Cipolla	onion	2366	ONION	697	697	627	240	633	100	156
+DellaRosa-2010-417-255	255	Circo	circus			616	660	453	228	626	179	205
+DellaRosa-2010-417-256	256	Collina	hill	614	HILL	674	665	595	261	621	150	375
+DellaRosa-2010-417-257	257	Coltello	knife	1352	KNIFE	697	700	680	230	675	110	138
+DellaRosa-2010-417-258	258	Commerciante	merchant	1082	MERCHANT	650	574	559	346	630	147	363
+DellaRosa-2010-417-259	259	Conchiglia	shell	598	SHELL	697	694	527	240	642	100	144
+DellaRosa-2010-417-260	260	Conte	earl			558	573	371	356	504	196	435
+DellaRosa-2010-417-261	261	Corallo	coral	3179	CORAL ROCK	650	638	441	358	570	144	369
+DellaRosa-2010-417-262	262	Corona	crown	1943	CROWN (OF HEAD)	668	647	488	242	585	153	300
+DellaRosa-2010-417-263	263	Cravatta	tie	3711	TIE (NOUN)	694	688	552	250	675	105	200
+DellaRosa-2010-417-264	264	Cristallo	crystal			661	584	439	374	581	165	332
```

Comment:
Maybe a good candidate for a concept. Few entries in previous lists present on Concepticon.